dashboard/scripts/archive/extra_level_files_for_dsl_defined_levels

```diff
@@ -0,0 +1,15 @@
+#!/usr/bin/env ruby
+# Find and remove extra .level files for dsl defined levels
+require_relative '../../config/environment'
+
+DSL_TYPES = %w(TextMatch ContractMatch External Match Multi)
+DSL_TYPES.map(&:constantize).each do |c|
+  c.all.each do |level|
+    # these should be dsl levels and should not have .level files
+    level_file = LevelLoader.level_file_path(level.name)
+    if File.exist? level_file
+      puts `ls -l '#{level_file}'`
```

Comment:
If this is something we will do a lot, perhaps add a --dry_run option, or make the script interactive (show the files and require confirmation before actually blowing them away)?